sql/create_tables.sql

```diff
@@ -0,0 +1,25 @@
+START TRANSACTION;
+
+DROP TABLE IF EXISTS car_prices;
+
+CREATE TABLE car_prices (
+    year INT,
+    make VARCHAR(255),
+    model VARCHAR(255),
+    trim VARCHAR(255),
+    body VARCHAR(255),
+    transmission VARCHAR(255),
+    vin VARCHAR(255),
+    state VARCHAR(255),
```

Comment:
now, when the data is preprocessed, we can use `VARCHAR(2)` for state
```suggestion
    state VARCHAR(2),
```